mmgen/models/architectures/vqgan/modules.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn.bricks import (Linear, build_activation_layer, build_conv_layer,
+                             build_norm_layer)
+from mmgen.registry import MODULES
+
+
+def nonlinearity(x):
+    """Sigmoid function.
+
+    Args:
+        x (torch.Tensor): Input feature map.
+
+    Returns:
+        torch.Tensor: Output feature map.
+    """
+    activate = build_activation_layer(dict(type='Sigmoid'))
+    return x * activate(x)
+
+
+def Normalize(in_channels):
```

Comment:
We do not need to add an extra function here. Just use build_norm_layer in your code.

Reply:
norm_cfg can be set as an argument.